### PR TITLE
Custom Start Time

### PR DESCRIPTION
**Pull request by BobbyTheCatfish**
Allow users to create a date object for the time they started playing

### CODE REVIEW

**Comment by E-boi** [comment]
Not sure if it works for but this is what shows for me 
image

**Comment by BobbyTheCatfish** [comment]
That's strange. I ran into that when I had the variable set to the wrong thing, but I thought I fixed that

**Comment by BobbyTheCatfish** [comment]
Oh, I think the plugin has to restart in order for the time to update

**Comment by E-boi** [comment]
I have restart multiple times, what is your custom time?

**Comment by BobbyTheCatfish** [comment]
I changed it to be time in minutes instead of a date object

File: index.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module.exports = class customRPC extends Plugin {
 			state: this.settings.get('state', 'Powercord Client'),
 			timestamps: this.settings.get('show_time', true)
 				? {
-						start: new Date(this.settings.get('start_time', undefined)) || Date.now(),
+						start: new Date(new Date().getTime() - this.settings.get('start_time', undefined)*60000) || Date.now(),
 				  }
 				: undefined,
```
Comment (E-boi) [inline index.js]: if you change it to this is becomes more responsive 
```JavaScript
timestamps: this.settings.get('show_time', true)
	? {
		start: this.settings.get('start_time', undefined)
		? new Date(new Date().getTime() - this.settings.get('start_time', undefined) * 60000).getTime()
		: Date.now(),
}
: undefined
```

Comment (E-boi) [reply index.js]: yo @BobbyTheCatfish if you make this small change i will the merge this pr

**Pull request merged**